test/parse.js

```diff
@@ -53,4 +53,13 @@ test('dates', function() {
             cookie.parse('priority=true; expires=Wed, 29 Jan 2014 17:43:25 GMT; Path=/',{
                 decode: function(value) { return value; }
             }));
-})
+});
+
+test('assign only once', function() {
+    assert.deepEqual({ foo: '%1', bar: 'bar' },
+        cookie.parse('foo=%1;bar=bar;foo=boo;HttpOnly;Secure'));
```

Comment:
Please remove the `HttpOnly;Secure` part from these strings, as that's not part of the `Cookie` header.